.github/workflows/crdgen.yaml

```diff
@@ -31,5 +31,9 @@ jobs:
       run: make -C crd/multitenantnetworkcontainer
     - name: Regenerate Multitenancy CRDs
       run: make -C crd/multitenancy
+    - name: Regenerate ClusterSubnetState CRD
+      run: make -C crd/clustersubnetstate
+    - name: Regenerate OverlayExtensionConfig CRD
+      run: make -C crd/overlayextensionconfig
```

Comment:
Is this something new with 1.23? Or should this be its own PR so we can backport it?

Reply:
it's fine if these are only in master, we just should have had the CI checking them already